### ISSUE TITLE
Switch to using Jason

### ISSUE THREAD

**Issue by Adzz** [googleapis/elixir-google-api#3415]
Currently there is an issue with Poison in this library.

Is it possible to switch to using [jason](https://github.com/michalmuskala/jason). It is faster, has become the default for lots of elixir libraries like Phoenix etc, and seems to be more stable. Not sure how much work that would be but thought it is worth asking.

**Comment by dazuma** [comment]
Yes, we probably should. Poison has been causing a lot of headache.

**Comment by sushant12** [comment]
any roadmap on switching?
